starter-code/app.js

```diff
@@ -1,5 +1,58 @@
 const express = require('express');
-const app = express();
 const Chuck  = require('chucknorris-io');
+
+const app = express();
 const client = new Chuck();
 
+//configure app
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
```

Comment:
It is a very good idea to have a route for /, if you want to display the same thing that in /random you can do:
```
app.get("/", (req, res) => {
  res.redirect('/random');
});
```